src/main/java/seedu/weme/model/util/SampleDataUtil.java

```diff
@@ -74,6 +78,20 @@ public static ReadOnlyMemeBook getSampleMemeBook(ReadOnlyUserPrefs userPrefs) {
         return sampleMb;
     }
 
+    public static StatsEngine getSampleStatsData(ReadOnlyUserPrefs userPrefs) {
+        StatsEngine statsEngine = new StatsManager();
+        statsEngine.setLikeData(getSampleLikeData(userPrefs));
+        return statsEngine;
+    }
+
+    public static LikeData getSampleLikeData(ReadOnlyUserPrefs userPrefs) {
```

Comment:
do we actually need a sample like data? Doesn't likedata work when it's an empty hashmap as well?

Reply:
True. Can actually remove as it's redundant.